content/src/roadmaps/license-tasks/apply-for-food-truck-license.md

```diff
@@ -20,6 +20,8 @@ summaryDescriptionMd: >
   You'll need to check license requirements with the county and/or local Department of Health in every county where you plan to do business.
 ---
 
-> **Once completed you will have:**
->
-> - A Department of Health license
+:::callout{ header="" icon="false" calloutType="conditional" }
```

Comment:
It would be cool to be a bit more consistent about the undefined vs empty string for the header field. Just because its nice and they both look like they're trying to invoke the default.